ip-quality-score/info.json

```diff
@@ -0,0 +1,405 @@
+          "description": "When this parameter is enabled our API will not perform an SMTP check with the mail service provider, which greatly increases the API speed. "
+        },
+        {
+          "title": "Timeout",
+          "name": "timeout",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "integer",
+          "value": 7,
+          "tooltip": "Provides better accuracy if response time is not an issue, timeout can be a value between 1 and 60. Default timeout is 7 seconds.",
+          "description": "Provides better accuracy if response time is not an issue, timeout can be a value between 1 and 60. Default timeout is 7 seconds."
+        },
+        {
+          "title": "Suggest Domain",
+          "name": "suggest_domain",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "checkbox",
+          "value": false,
+          "tooltip": "Force analyze if the email address's domain has a typo and should be corrected to a popular mail service. By default, this test is currently only performed when the email is invalid or if the \"recent abuse\" status is true.",
+          "description": "Force analyze if the email address's domain has a typo and should be corrected to a popular mail service. By default, this test is currently only performed when the email is invalid or if the \"recent abuse\" status is true."
+        },
+        {
+          "title": "Strictness",
+          "name": "strictness",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "options": [
+            "0",
+            "1",
+            "2"
+          ],
+          "value": [
+            "0"
+          ],
+          "type": "text",
+          "tooltip": "Sets how strictly spam traps and honeypots are detected by our system, depending on how comfortable you are with identifying emails suspected of being a spam trap. 0 is the lowest level which will only return spam traps with high confidence. Strictness levels above 0 will return increasingly more strict results, with level 2 providing the greatest detection rates.",
+          "description": "Sets how strictly spam traps and honeypots are detected by our system, depending on how comfortable you are with identifying emails suspected of being a spam trap. 0 is the lowest level which will only return spam traps with high confidence. Strictness levels above 0 will return increasingly more strict results, with level 2 providing the greatest detection rates."
+        },
+        {
+          "title": "Abuse Strictness",
+          "name": "abuse_strictness",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "options": [
+            "0",
+            "1",
+            "2"
+          ],
+          "value": [
+            "0"
```

Comment:
here api accept string or integer?

Reply:
Modified the info.json and convert string to int